omod/src/main/java/org/openmrs/module/queue/web/resources/QueueResource.java

```diff
@@ -156,6 +166,16 @@ public String getDisplay(Queue queue) {
 		return queue.getName();
 	}
 	
+	@PropertyGetter("allowedPriorities")
+	public Object getAllowedPriorities(Queue delegate) {
+		return services.getAllowedPriorities(delegate);
+	}
+	
+	@PropertyGetter("allowedStatuses")
+	public Object getAllowedStatuses(Queue delegate) {
+		return services.getAllowedStatuses(delegate);
+	}
```

Comment:
I'm not entirely sure if this is the right way to support this, but adding these to the Queue resource seems the most straightforward and helpful.